src/studio-header/StudioHeader.test.jsx

```diff
@@ -128,6 +128,13 @@ describe('Header', () => {
       expect(avatarIcon).toBeVisible();
     });
 
+    it.only('user menu should not contain username', async () => {
+      const newProps = { ...props, ENABLE_HEADER_WITHOUT_USERNAME: true };
```

Comment:
Why are we sending the environment variable as a prop?

Reply:
Mistakenly added it here, We are enabling this flag using mergeConfig.